prac_05/kivy_layout.kv

```diff
@@ -1,4 +1,4 @@
-# This is a Kv file.
+# This is a Kv file
```

Comment:
Line 23 - Should the size be 0.2 if the label is required to take up 10% of the vertical space?